crates/rolldown/tests/rolldown/function/hashing/maximum-hash-size/artifacts.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+---
+# Errors
+
+## UNHANDLEABLE_ERROR
+
+```text
+[UNHANDLEABLE_ERROR] Error: Something went wrong inside rolldown, please report this problem at https://github.com/rolldown/rolldown/issues.
```

Comment:
We should emit a different type of error because this message is misleading.
Rollup emits `VALIDATION_ERROR`, but I guess it can also be `INVALID_OPTION`.